shell/platform/fuchsia/flutter/component.cc

```diff
@@ -105,6 +110,16 @@ static std::unique_ptr<fml::FileMapping> MakeReadExecMapping(const char* path) {
   return mapping;
 }
 
+// Defaults to readonly. If executable is `true`, we treat it as `read + exec`.
+static flutter::MappingCallback MakeDataFileMapping(const char* file_name,
+                                                    bool executable = false) {
+  std::string absolute_path = "/pkg/data/";
```

Comment:
Lets just make the caller pass us the full path. This method doesn't need to have an opinion about where the files live.

Reply:
sg